### ISSUE TITLE
Empty Buttons keyboard navigation

### ISSUE THREAD

**Issue by jmn90** [FrDH/mmenu-js#1222]
Hi, I know an issue have already been open about this subject in the past https://github.com/FrDH/mmenu-js/issues/817

But it seems that there is still an accessibility issue to be resolved unless there are some workarounds that I am not aware of. 

The problem I noticed is that the two empty buttons are still tabbable, meaning that a user tabbing through the menu have no idea what the purpose of those buttons are. Especially a keyboard user that doesn't need a screen reader since they don't know where the focus went.

So would it be possible to do one of the following : 

- Dynamically find the first and last tabbable element and use it for the focus trap
- Add an option to specify customs tabstart and tabend elements
- Add an option to specify the first element that should be focused and make the tabstart and tabend visible on focus with a class like "mm-sronly-focusable" as well. That way keyboard users would see the buttons when they tab on it and mouse/touch users would not be affected.

Thanks!

**Comment by FrDH** [comment]
Hi, 
Thank you for the feedback, keyboard navigation will be improved in the next major release.
The "mm-sronly-focusable" class is a nice addition, I'll keep that in mind.

**Comment by FrDH** [comment]
The next major release will have better accessibility implementations. Please keep an eye out for it.

**Comment by ikonplant** [comment]
Hi @FrDH !

When can we expect the next major release and the fix of this issue?
Is there already any commit solving this problem?

Thank you for information.
